pkgs/applications/misc/k40-whisperer/default.nix

```diff
@@ -46,14 +58,21 @@ in stdenv.mkDerivation rec {
 
     mkdir -p $out/bin
     mkdir -p $out/lib/udev/rules.d
+    mkdir -p $out/share/icons/hicolor/128x128/apps
+    mkdir -p "$out/share/applications"
 
     ln -s ${udevRule} $out/lib/udev/rules.d/97-k40-whisperer.rules
+    ln -s "${desktopItem}/share/applications/k40-whisperer.desktop" "$out/share/applications/k40-whisperer.desktop"
+
+    gm convert "$out/scorchworks.ico" "$out/share/icons/hicolor/128x128/apps/k40-whisperer.png"
 
     makeWrapper '${pythonEnv.interpreter}' $out/bin/k40-whisperer \
       --add-flags $out/k40_whisperer.py \
       --prefix PYTHONPATH : $out
   '';
 
+  desktopItems = [ desktopItem ];
```

Comment:
This should be unused

Reply:
What do you mean? Should this line be removed?

Reply:
sorry, I can't remember